packages/namekit-react/src/components/DisplayedPrice.tsx

```diff
@@ -35,62 +12,39 @@ export const PriceDisplaySize = {
 export type PriceDisplaySize =
   (typeof PriceDisplaySize)[keyof typeof PriceDisplaySize];
 
-/**
- * A mapping from a `PriceDisplaySize` to a `CurrencySymbolSize`.
- * This is useful for determining the size of the currency symbol to display
- * for a given price display size. e.g. as needed in DisplayedPrice, when a
- * currency symbol is displayed alongside a price and the user can define
- * the size of the price display. This way both price display size and
- * currency symbol size can be controlled by the same prop, always
- * guaranteeing that they are displayed with aligned sizes.
- * @param priceDisplaySize: PriceDisplaySize. The size of the price display.
- * @returns CurrencySymbolSize. The size of the currency symbol to display.
- */
-export const getCurrencySymbolSize = (
-  priceDisplaySize: PriceDisplaySize,
-): CurrencySymbolSize => {
-  switch (priceDisplaySize) {
-    case PriceDisplaySize.Micro:
-    case PriceDisplaySize.Small:
-      return CurrencySymbolSize.Small;
-    case PriceDisplaySize.Medium:
-    case PriceDisplaySize.Large:
-      return CurrencySymbolSize.Large;
-  }
-};
-
 export interface DisplayedPriceProps {
-  // The price to be displayed
+  /**
+   * The price to be displayed
+   * @example { currency: Currency.Eth, value: 1000000000000000000n }
+   */
   price: Price;
-  // Wether to display a currency symbology as its acronym or symbol
-  currencySymbology?: PriceSymbology;
+  /**
+   * The currency symbol to display alongside the price.
+   */
+  symbol: React.ReactNode;
   /**
    * The position of the currency symbol relative to the price.
    * Defaults to `CurrencySymbolPosition.Left`.
    *
    * If `CurrencySymbolPosition.Left`, the currency symbol will be displayed to the left of the price.
    * If `CurrencySymbolPosition.Right`, the currency symbol will be displayed to the right of the price.
    */
-  symbolPosition?: CurrencySymbolPosition;
   // The size of the price display
   displaySize?: PriceDisplaySize;
-  // Wether or not to display the name of the currency in a tooltip when its symbol is hovered
-  describeCurrencyInTooltip?: boolean;
-  // Whenever below prop is defined, the price is displayed inside a Tooltip which is triggered by the below content
+  /**
+   * Whenever below prop is defined, the price will be
+   * displayed inside a Tooltip and the below content
+   * will be the trigger of this Tooltip displaying.
+   */
   tooltipTriggerToDisplayPriceInTooltip?: React.ReactNode;
 }
 
 export const DisplayedPrice = ({
   price,
-  describeCurrencyInTooltip = true,
+  symbol,
   displaySize = PriceDisplaySize.Small,
   tooltipTriggerToDisplayPriceInTooltip,
-  currencySymbology = PriceSymbology.Symbol,
-  symbolPosition = CurrencySymbolPosition.Left,
```

Comment:
@FrancoAguzzi Suggest we still keep this idea in `DisplayedPrice`. Please see how `IconButton` includes a prop for this idea and how they implemented it.